Automattic/simplenote-macos
# Shortcuts: Open Note

### Fix
Okay, so back on track after a bunch of setup PRs.  What we have here is the ability to open a note from a shortcut.  That should be fairly simple, but there were a lot of core technologies that needed to be added to do this.  So sorry but this PR is a little long.  One the plus side several hundred lines of it are all xml files that are auto generated and shouldn't need much review.

Anyways, so the shortcut that I have added here allows you to specify or search for a note and then open it in Simplenote.  Straight forward, but it is defining some of the core tech we will need for more advanced shortcuts.

Moves forward: #1145 
relies on #1162 

### Test
1. build and launch Simplenote.  Make sure you are logged in
2. Open the shortcuts app, create a new shortcut, find simplenote, select Open Note
3. In the shortcut that gets created select a note to open
4. Run the shortcut.  Simplenote should launch and open that note.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> These changes do not require release notes.
